packages/neuron-ui/src/widgets/DatetimePicker/index.tsx

```diff
@@ -155,14 +129,7 @@ const DatetimePicker = ({
             onKeyPress={onKeyPress}
           />
         )}
-        <Calendar
-          value={selected}
-          minDate={new Date()}
-          onChange={onCalendarChange}
-          inline
-          locale="es"
-          className={styles.calendar}
-        />
+        <Calendar value={selected} minDate={tomorrow} onChange={onCalendarChange} lang={i18n.language} />
```

Comment:
The `language` could be fetched by `useTranslation` instead of a global instance.

Besides, the `language` could be fetched inside `Calendar`

Ref: https://github.com/nervosnetwork/neuron/blob/1f19cca93b3247447ab6082eca16021dd6656db6/packages/neuron-ui/src/containers/Settings/index.tsx#L43